contract-ui/tabs/overview/page.tsx

```diff
@@ -44,11 +46,30 @@ export const CustomContractOverviewPage: React.FC<
   return (
     <SimpleGrid columns={{ base: 1, xl: 4 }} gap={8}>
       <GridItem as={Flex} colSpan={{ xl: 3 }} direction="column" gap={16}>
+        {contract &&
+          ["ERC1155", "ERC721"].some((type) =>
+            detectedFeatureNames.includes(type),
+          ) && (
+            <NFTDetails
+              contract={contract}
+              trackingCategory={TRACKING_CATEGORY}
+              features={detectedFeatureNames}
+            />
+          )}
         <LatestEvents
           address={contractAddress}
           trackingCategory={TRACKING_CATEGORY}
         />
         <BuildYourApp trackingCategory={TRACKING_CATEGORY} />
+        {contract &&
+          ["Permissions", "PermissionsEnumerable"].some((type) =>
```

Comment:
Let's only use the enumerable version for now, for basic "permissions" you're not going to be able to ask the contract for a list of people with roles so this will have to be a different type of UI